Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.h

```diff
@@ -149,8 +149,7 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLModelDisplayableManager
   static vtkLookupTable* CreateLookupTableCopy(vtkLookupTable* source);
 
   /// Helper function for determining what type of scalar is active.
-  static bool IsCellScalarsActive(vtkMRMLDisplayNode* displayNode,
-    vtkMRMLModelNode* model = 0);
+  static bool IsCellScalarsActive(vtkMRMLDisplayNode* displayNode, vtkMRMLModelNode* model = 0);
```

Comment:
It may be sensible to update the docstring

Reply:
Not sure what you mean. I just removed a line break from here, nothing changed.

Reply:
It could have been the opportunity to improve the docstring. It also fine to leave things the way they are.